index.d.ts

```diff
@@ -417,7 +493,7 @@ export interface CbExecutionContext<Context = unknown> extends ExecutionContext<
 	end(error?: any): void;
 }
 
-export type ImplementationResult = PromiseLike<void> | Subscribable | void;
+export type ImplementationResult = PromiseLike<void> | Subscribable | boolean | void;
```

Comment:
`npx tsd` fails with more than a dozen errors about the type of `ImplementationResult` if this is not here. Any ideas on alternatives @novemberborn ?

Reply:
This is a problems with the type definitions. It's fine for now, I'll address this separately for AVA 4.